src/algorithms/reco/ChargedMCParticleSelector.h

```diff
@@ -0,0 +1,45 @@
+// SPDX-License-Identifier: LGPL-3.0-or-later
+// Copyright (C) 2023 Dmitry Kalinkin
```

Comment:
```suggestion
// Copyright (C) 2023, 2024, Dmitry Kalinkin, Derek Anderson
```

Reply:
Something tells me there is a pre-commit checker for spdx copyright lines...